pkg/apis/messaging/v1beta1/subscription_conversion_test.go

```diff
@@ -19,9 +19,31 @@ package v1beta1
 import (
 	"context"
 	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+	eventingduckv1 "knative.dev/eventing/pkg/apis/duck/v1"
+	duckv1beta1 "knative.dev/eventing/pkg/apis/duck/v1beta1"
+	v1 "knative.dev/eventing/pkg/apis/messaging/v1"
```

Comment:
`messagingv1`

Reply:
or remove the renaming

Reply:
If you don't feel too strong about it, let's leave it as `v1`.
This test is about converting subscriptions from v1beta1 to v1.  Name v1 is more understandable IMO as we're already in messaging v1beta1 package.

Reply:
matmoor's bot suggested to add v1, so I added the name it :)

Reply:
Agree, but then we don't need to rename it, right?

Reply:
Now, I saw a similar pattern for the other conversion and there we don't include `messaging*` or `eventing*`, sorry for the noise :)